geofileops/geoops.py

```diff
@@ -106,6 +106,8 @@ def dissolve_within_distance(
             return
         else:
             fileops.remove(output_path)
+    elif not output_path.parent.exists():
+        raise ValueError(f"{operation_name}: output_path doesn't exist")
```

Comment:
```suggestion
        raise ValueError(f"{operation_name}: output_path directory doesn't exist")
```
I think it will be clearer if it is mentioned that the directory doesn't exist, because it is normal that the full `output_path` doesn't exist?